src/database/queries/Collection.ts

```diff
@@ -94,6 +96,7 @@ export async function getCollectionsBySlugs(
       curationCategory: true,
       IABChildCategory: true,
       IABParentCategory: true,
+      labels: true,
       partnership: true,
```

Comment:
These are the actual code changes. Everything else is new/updated tests + typo fixes here and there.